Doc/whatsnew/3.9.rst

```diff
@@ -166,6 +166,13 @@ ftplib
 if the given timeout for their constructor is zero to prevent the creation of
```

Comment:
@pablogsal the file Doc/myfile.bz2 above does not seem referenced from anywhere.

Reply:
That has been corrected since this PR :)

Reply:
Particulary in 67673b08ea302174ca9c6559c04311550ae9f189